src/styles/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  gap: 1px;
+  justify-content: left;
+  width: 706px;
+  padding: 10px;
+  flex-wrap: wrap;
+
+  & :nth-child(n) {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 100px;
+    min-height: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+  }
+
+  & :nth-child(n):hover {
```

Comment:
Use nesting:

This can be nested inside :nth-child(n):
```
    &:nth-child(n) {
        ...your styling goes here

        &:hover {
            ...hovered style goes here
        }
    }
```